src/main/resources/ValidationMessages.properties

```diff
@@ -7,3 +7,4 @@ otherAccountingPolicy.selectionNotMade=Tell us if your prepared accounts include
 valuationInformationPolicy.selectionNotMade=Tell us if your prepared accounts include a ''valuation information'' policy
 corporationTax.selectionNotMade=Tell us if you also want to file your corporation tax with HMRC now
 typeOfAccounts.selectionNotMade=Tell us which type of accounts you have prepared
+fixedAssetsInvestments.details.missing = Provide details of your 'fixed assets investments'
```

Comment:
Will this message need double single-quotes like the others?

Reply:
Yes, it should have.  Double single quotes added.